config/config.go

```diff
@@ -339,6 +339,14 @@ type RPCConfig struct {
 	// global HTTP write timeout, which applies to all connections and endpoints.
 	// See https://github.com/tendermint/tendermint/issues/3435
 	TimeoutBroadcastTxCommit time.Duration `mapstructure:"timeout_broadcast_tx_commit"`
+
+	// The name of cert file that used to serve RPC based on Tls.
+	// NOTE: useful when tls_key_file is present
+	TlsCertFile string `mapstructure:"tls_cert_file"`
+
+	// The name of private key file that used to serve RPC based on Tls.
```

Comment:
```
// The name of a file containing matching private key that is used to create the HTTPS server.
//
// NOTE: both tls_cert_file and tls_key_file must be present for Tendermint to create HTTPS server. Otherwise, HTTP server is run.
```